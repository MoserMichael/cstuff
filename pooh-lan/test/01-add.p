# mixing strings with numers is allowed in arithmetic operators
a = '22'
b = 3

c = a + b

println( ~msg c )
