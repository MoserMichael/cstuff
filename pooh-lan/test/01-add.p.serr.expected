002|a = '22'
003|b = 3
005|c = (a:'22' + b:3):2.500000e+01
007|println( ~msg c:2.500000e+01 )...
